tests/testthat/test_retentionIndexing.R

```diff
@@ -0,0 +1,17 @@
+test_that("retention indexing linear works", {
+  
+  rti <- read.table(system.file("retentionIndex/rti.txt",
+                                package = "MetaboCoreUtils"),
+                    header = TRUE,
+                    sep = "\t")
+  
+  rtime <- read.table(system.file("retentionIndex/metabolites.txt",
+                                  package = "MetaboCoreUtils"),
+                      header = TRUE,
+                      sep = "\t")
+  
+  rtime$rindex2 <- rtimeIndexing(rtime$rtime, rti)
+  
+  expect_equal(round(rtime$rindex_manual, 0), round(rtime$rindex2, 0))
+
```

Comment:
could you maybe also add some unit tests that check what happens if `x` or `y` are empty, or if `y` does not have the required columns?

Reply:
Done.